project-keeper/src/main/java/com/exasol/projectkeeper/validators/files/ProjectFilesValidator.java

```diff
@@ -103,58 +106,77 @@ public static Builder builder() {
      * Builder for new instances of {@link ProjectFilesValidator}
      */
     public static final class Builder {
-        private final ProjectFilesValidator validator = new ProjectFilesValidator();
+        private Path projectDirectory;
```

Comment:
:+1: You caught the lazy _builder_ created by me.
Sorry :slightly_frowning_face: